### ISSUE TITLE
Jira Issue Url To Tags Feature Not Working

### ISSUE THREAD

**Issue by mkubarycz** [marc0l92/obsidian-jira-issue#24]
The jira issue url to tags feature does not appear to work which leads to me needing to manually convert a jira url to a inline prefix style notation (ie: jira:XX-123)

Steps to reproduce:

- Turn on "Issue url to tags" and set inline issue prefix to "jira:"
- Open daily note and type the following...

```
jira:XX-6765
https://XXX.atlassian.net/browse/XX-6765
jira:https://XXX.atlassian.net/browse/XX-6765
```

- **Observe that only the first line gets converted to a jira tag.  I expected all three to get converted.**

**Comment by marc0l92** [comment]
Thanks for reporting this issue.
I'm aware of this limitation. At the moment this feature is available only in reading mode and not in live preview.

If you find it useful, I can implement it for live preview as well.

The syntax will be like the one in the second row of your example.

**Comment by marc0l92** [comment]
Hello, you can now use this feature in live preview mode.
Use the following syntax to add an extended tag:
```
https://my-company.atlassian.net/browse/ABC-123
```
prepend the link with a `-` symbol to have the compact tag:
```
-https://my-company.atlassian.net/browse/ABC-123
```

The feature is available starting from 1.37.0

**Comment by dansteen** [comment]
Thanks @marc0l92 -  I was going to open a ticket for this as well! I really appreciate your work on this plugin, it has **really** enhanced my workflow!
